src/v2/view-builder/views/DeviceEnrollmentView.js

```diff
@@ -0,0 +1,80 @@
+import { createCallout, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+
+const ODA = 'oda';
+const MDM = 'mdm';
+const IOS = 'ios';
+const ANDROID = 'android';
+
+const Body = BaseForm.extend({
+  noButtonBar: true,
+
+  title () {
+    return this.enrollmentType === ODA ?
+      'Download Okta Verify' : 'Additional setup required';
+  },
+
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    const deviceEnrollment = this.options.appState.get('deviceEnrollment');
+    this.enrollmentType = (deviceEnrollment.name || '').toLowerCase(); // oda/mdm
+    switch (this.enrollmentType) {
+    case ODA:
+      this.add(View.extend({
```

Comment:
I would extract these views from here and assign them as constant before we extend BaseForm